index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Spring | Projects</title>
+    <link rel="stylesheet" href="style.css"/>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+    <title>Document</title>
+</head>
+<body>
+       <header>
+         <nav id="nav-menu">
```

Comment:
Class naming is not consistent (`nav-menu`, `menu_items`,  `dropdownMenu`). Please use one style for all class names